gnosis/verification.k

```diff
@@ -67,14 +81,17 @@ module VERIFICATION
 
     rule WS ++ .WordStack => WS
 
+    rule sizeWordStackAux(WS, N) <Int SIZE => sizeWordStackAux(WS, 0) +Int N <Int SIZE  requires N =/=Int 0
+    rule SIZELIMIT <Int sizeWordStackAux(WS, N) +Int DELTA  => SIZELIMIT <Int (sizeWordStackAux(WS, 0) +Int N) +Int DELTA  requires N =/=Int 0
+    rule SIZELIMIT <Int sizeWordStackAux(WS, N) => SIZELIMIT <Int sizeWordStackAux(WS, 0) +Int N  requires N =/=Int 0
+
+    rule #sizeWordStack(selectRange(_, _, WIDTH), SIZE) => SIZE +Int WIDTH
+
 
   // ########################
   // Range
   // ########################
 
-    //todo should not be needed since #bufSeg(_,_,1) is converted to #bufElm
-    rule #asWord(#bufSeg(_, _, WIDTH)) <Int 256 => true requires WIDTH ==Int 1
```

Comment:
are you sure this is not needed? or any particular reason not to have this?

Reply:
``` 
    rule #bufSeg(BUF, START, 1) => #bufElm(BUF, START) : .WordStack
    rule #asWord(#bufElm(BUF, INDEX) : .WordStack) => #bufElm(BUF, INDEX)
    rule #bufElm(_, _) <Int 256 => true
```

Reply:
```
   rule #bufSeg(BUF, START, 1) => #bufElm(BUF, START) : .WordStack
    rule #asWord(#bufElm(BUF, INDEX) : .WordStack) => #bufElm(BUF, INDEX)
    rule #bufElm(_, _) <Int 256 => true
```

Reply:
ok.